plugins/trayicon.py

```diff
@@ -26,6 +26,18 @@
 from gi.repository import GObject, Gtk, Liferea
 from gi.repository import Gdk, GdkPixbuf
 
+gi.require_version('AyatanaAppIndicator3', '0.1')
```

Comment:
The gi.require not being safe-guarded by the `try` below causes a 

     ValueError: Namespace AyatanaAppIndicator3 not available

for users without Ayatana GIR bindings. So it needs to be included in the `try` and both `except` need to be extended to

     except (ImportError, ValueError)

Reply:
I pushed the fix. Thanks!